custom-metrics-stackdriver-adapter/pkg/adapter/translator/response_translator.go

```diff
@@ -243,6 +248,10 @@ func (t *Translator) metricKey(timeSeries *stackdriver.TimeSeries) (string, erro
 		case "k8s_node":
 			return ":" + timeSeries.Resource.Labels["node_name"], nil
 		case "prometheus_target":
+			if resourceSchema == "nodes" {
+				// ignore port number part
```

Comment:
can you add an example for what this is about?

Reply:
yep, for `k8s_node` type metrics, usually it's just the node id (`gke-stackdriver-adapter--default-pool-cee13989-0i75`), but for `prometheus_target` metrics, it will also include the port number who emits the metric (gke-stackdriver-adapter--default-pool-cee13989-0i75:8080).

I can add the explanation to the comment as well